ReactWindows/ReactNative.Shared/UIManager/BorderedViewParentManager.cs

```diff
@@ -118,137 +149,37 @@ public void SetBorderWidth(Border view, int index, double width)
-        /// </summary>
-        /// <param name="reactContext">The React context.</param>
-        /// <returns>The view instance.</returns>
-        protected sealed override Border CreateViewInstance(ThemedReactContext reactContext)
-        {
-            var inner = CreateInnerElement(reactContext);
-            return new Border
-            {
-                BorderBrush = s_defaultBorderBrush,
-                Child = inner
-            };
-        }
-
-        /// <summary>
-        /// Creates a new view instance of type <typeparamref name="TFrameworkElement"/>.
-        /// </summary>
-        /// <param name="reactContext">The React context.</param>
-        /// <returns>The view instance.</returns>
-        protected abstract TFrameworkElement CreateInnerElement(ThemedReactContext reactContext);
-
-        /// <summary>
-        /// Adds a child at the given index.
-        /// </summary>
-        /// <param name="parent">The parent view.</param>
-        /// <param name="child">The child view.</param>
-        /// <param name="index">The index.</param>
-        protected abstract void AddView(TFrameworkElement parent, DependencyObject child, int index);
-
-        /// <summary>
-        /// Gets the number of children in the view parent.
-        /// </summary>
-        /// <param name="parent">The view parent.</param>
-        /// <returns>The number of children.</returns>
-        protected abstract int GetChildCount(TFrameworkElement parent);
-
-        /// <summary>
-        /// Gets the child at the given index.
-        /// </summary>
-        /// <param name="parent">The parent view.</param>
-        /// <param name="index">The index.</param>
-        /// <returns>The child view.</returns>
-        protected abstract FrameworkElement GetChildAt(TFrameworkElement parent, int index);
-
-        /// <summary>
-        /// Removes the child at the given index.
-        /// </summary>
-        /// <param name="parent">The view parent.</param>
-        /// <param name="index">The index.</param>
-        protected abstract void RemoveChildAt(TFrameworkElement parent, int index);
-
-        /// <summary>
-        /// Removes all children from the view parent.
-        /// </summary>
-        /// <param name="parent">The view parent.</param>
-        protected abstract void RemoveAllChildren(TFrameworkElement parent);
-
-        /// <summary>
-        /// Get the inner element of the border.
-        /// </summary>
-        /// <param name="parent">The parent view.</param>
-        /// <returns>The inner element.</returns>
-        protected TFrameworkElement GetInnerElement(Border parent)
-        {
-            if (parent == null)
-                throw new ArgumentNullException(nameof(parent));
-
-            return (TFrameworkElement)parent.Child;
+            var pointerEvents = EnumHelpers.ParseNullable<PointerEvents>(pointerEventsValue) ?? PointerEvents.Auto;
+            view.SetPointerEvents(pointerEvents);
         }
```

Comment:
Ah, yeah these two props should be on ReactViewManager (they currently are in the master branch of react-native-windows).

---
In reply to: [127970487](https://github.com/rozele/react-native-windows/pull/1#discussion_r127970487) [](ancestors = 127970487)